modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/messaging/MonitoringMessageListener.java

```diff
@@ -192,6 +185,12 @@ protected synchronized void unregisterDataSampleProcessor(
 
 	private final Map<String, List<DataSampleProcessor<DataSample>>>
 		_dataSampleProcessors = new ConcurrentHashMap<>();
+
+	@Reference(
+		target = "(destination.name=" + DestinationNames.MONITORING + ")"
+	)
+	private Destination _destination;
```

Comment:
@W2jx You do not need this dependency, please remove. Even if the destination is not ready, the listener is activated and registered, you won't be able to use the listener as there is no destination connected to it.